pages/how-it-works.js

```diff
@@ -43,7 +43,7 @@ const HowItWorks = () => (
           Alternatively, if a donation is chosen as your method of giving back
           to the Open Source community, 97% of the donation (after processing
           fees) is passed along to the maintainers of the Open Source packages
-          installed. You can edit, delete, or up your donation at any time
+          installed and no ads will be shown. You can edit, delete, or up your donation at any time
```

Comment:
while you're here.... "edit, delete, or up" ? probably simpler to just say "you can change or delete your donation" or something like that. (up == edit)

Reply:
done